boards/arm/adafruit_feather_nrf52840/adafruit_feather_nrf52840_sense_defconfig

```diff
@@ -0,0 +1,30 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_NRF52X=y
+CONFIG_SOC_NRF52840_QIAA=y
+CONFIG_BOARD_ADAFRUIT_FEATHER_NRF52840_SENSE=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# enable uart driver
+CONFIG_SERIAL=y
+
+# enable console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# 32kHz clock source
+CONFIG_CLOCK_CONTROL_NRF_K32SRC_RC=y
+CONFIG_CLOCK_CONTROL_NRF_K32SRC_500PPM=y
+
+# enable bossac
+CONFIG_BOOTLOADER_BOSSA=y
+CONFIG_BOOTLOADER_BOSSA_ADAFRUIT_UF2=y
+
+# Build UF2 by default, supported by the Adafruit nRF52 Bootloader
+CONFIG_BUILD_OUTPUT_UF2=y
+CONFIG_USE_DT_CODE_PARTITION=y
```

Comment:
According to https://docs.zephyrproject.org/latest/develop/flash_debug/host-tools.html this final line should not be needed

> 2. Have board :file:`.defconfig` file with the :kconfig:option:`CONFIG_BOOTLOADER_BOSSA` Kconfig option set to ``y``.  This will automatically select the :kconfig:option:`CONFIG_USE_DT_CODE_PARTITION` Kconfig option which instruct the build system to use these partitions for code relocation.  The board :file:`.defconfig` file should have :kconfig:option:`CONFIG_BOOTLOADER_BOSSA_ARDUINO` , :kconfig:option:`CONFIG_BOOTLOADER_BOSSA_ADAFRUIT_UF2` or the :kconfig:option:`CONFIG_BOOTLOADER_BOSSA_LEGACY` Kconfig option set to ``y`` to select the right compatible SAM-BA bootloader mode. These options can also be set in ``prj.conf`` or any other Kconfig fragment.

Reply:
I'll remove it, thanks

Reply:
should I also remove it from adafruit_itsybitsy_nrf52840_defconfig?

Reply:
Can do, but do that in a separate PR